### PR TITLE
[HOPSWORKS-1667] featurestore index migration

### PR DESCRIPTION
**Pull request by o-alex**

### CODE REVIEW

File: docs/getting_started/installation_guide/upgrades/1_3_0.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+=============
+Version 1.3.0
+=============
+
+Upgrading to version 1.3.0 allows for the search of featuregroups and training dataset. In order for search to work on old featuregroups and training datasets, you need to enable reindexing of featurestore index in your upgrade recipe: elastic->featurestore->reindex: true. In order to complete the upgrade, you also need to run expat migration to 1.3.0 so that the necessary extended attributes are attached. The featurestore reindexing can also be done manually from epipe. The two operations: reindex and expat migration are commutative.
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/upgrades/1_3_0.rst:5]: `training datasets`

File: docs/getting_started/installation_guide/upgrades/1_3_0.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+There is currently a limitation, for large featuregroups and training datasets and if they contain too many component features, the features will not be searchable. Current limit is about 200 features per featuregroup/trainingdataset, if you want the featuregroup/training dataset to be discoverable by the component features as well.
```
Comment (tkakantousis) [inline docs/getting_started/installation_guide/upgrades/1_3_0.rst:7]: We can say the limitation is planned to be removed with the next version

File: docs/getting_started/installation_guide/upgrades/1_3_0.rst (hunk 1 above)
Comment (o-alex) [reply docs/getting_started/installation_guide/upgrades/1_3_0.rst]: fixed

File: docs/getting_started/installation_guide/upgrades/1_3_0.rst (hunk 2 above)
Comment (o-alex) [reply docs/getting_started/installation_guide/upgrades/1_3_0.rst]: updated to, upcoming version will support up to 50000 features per featuregroup and still have them discoverable.

**Pull request merged**